spec/integration/criteria/time_with_zone_spec.rb

```diff
@@ -4,29 +4,140 @@
 require 'spec_helper'
 
 describe 'TimeWithZone in queries' do
-  context 'in a non-UTC time zone' do
-    let(:time_zone_name) { 'Pacific Time (US & Canada)' }
+  let(:now_utc) { Time.now.utc }
+  let(:now_in_zone) { now_utc.in_time_zone(time_zone) }
 
-    before do
-      time = Time.now
-      expect(time.utc_offset).not_to eq(time.in_time_zone(time_zone_name).utc_offset)
+  shared_examples_for 'time zone queries' do
+    let!(:agency1) { Agency.create!(updated_at: now_utc - 20.minutes, dynamic_time: now_utc - 20.minutes).reload }
+    let!(:agency2) { Agency.create!(dynamic_time: now_in_zone).reload }
+    let!(:agency3) { Agency.create!(updated_at: now_in_zone + 20.minutes, dynamic_time: now_in_zone + 20.minutes).reload }
```

Comment:
Can these variables please be renamed to reflect their contents, e.g. `agency_utc` instead of `agency1`.

Reply:
Done --> book_earlier, book_now, and book_later